packages/codemods/nuxt/4/template-compilation-changes/.codemodrc.json

```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "https://codemod-utils.s3.us-west-1.amazonaws.com/configuration_schema.json",
+  "version": "1.0.0",
+  "private": false,
+  "name": "nuxt/4/shallow-function-reactivity",
+  "engine": "jscodeshift",
+  "applicability": {
+    "from": [["nuxt", ">", "3.0.0"]],
+    "to": [["vue", ">=", "4.0.0"]]
```

Comment:
ideally, the From should be a range, and To is just the first version where the new pattern is introduced.
also lets use >= if 3.0.0 should be included too.

![Screenshot 2024-06-22 at 4 34 03 PM](https://github.com/codemod-com/codemod/assets/78109534/23bfee4e-6340-4aaa-a170-c1611c192dd5)

Reply:
if the lower bound of From is not known, at least we need to provide the upper bound... in your example, you left the upper bound empty...
also why the second one says vue? :D should it say nuxt?